storybook/webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = async ({ config }) => {
   });
 
   config.module.rules.push({
-    test: /stories\/.*\.(ts|tsx)$/,
+    test: /stories(\\|\/).*\.(ts|tsx)$/,
```

Comment:
why not adding this fix to non-typescript story files as well? 
see line #5 on this file.

Reply:
Sorry I missed that. Just realized that sources are not there in Windows because of the other regex. 
I have changed this and  added a new commit

Reply:
Can you verify it fixes the issue fo you?
@markjackson02 can you verify as well? I think you encountered a similar issue.

Reply:
It fixes the issue for me

Reply:
It fixes the issue for me